docs/inputs/index.md

```diff
@@ -8,33 +8,33 @@ Developers are required to add services in the global config file to create a ne
 
 ### Properties
 
-| Property                                                                  | Type   | Description                                                                                           |
-| ------------------------------------------------------------------------- | ------ | ----------------------------------------------------------------------------------------------------- |
-| title<span class="required-asterisk">\*</span>                            | string | -                                                                                                     |
+| Property                                                                  | Type   | Description                                                                                            |
+| ------------------------------------------------------------------------- | ------ | ------------------------------------------------------------------------------------------------------ |
+| title<span class="required-asterisk">\*</span>                            | string | -                                                                                                      |
 | description                                                               | string | It provides a brief summary of an inputs page.                                                         |
+| [subDescription](../advanced/sub_description)                             | object | It provides broader description of an inputs page.                                                     |
```

Comment:
Since we are here, it would be nice to fix links by adding `.md` here. It will fix navigation inside Github Markdown previews and warnings from `mkdocs`:
```
INFO    -  Doc file 'inputs/index.md' contains an unrecognized relative link '../table', it was left as is. Did you mean '../table.md'?
INFO    -  Doc file 'inputs/index.md' contains an unrecognized relative link '../entity', it was left as is. Did you mean '../entity/index.md'?
INFO    -  Doc file 'inputs/index.md' contains an unrecognized relative link '../advanced/groups_feature', it was left as is. Did you mean '../advanced/groups_feature.md'?
INFO    -  Doc file 'inputs/index.md' contains an unrecognized relative link '../advanced/save_validator', it was left as is. Did you mean '../advanced/save_validator.md'?
INFO    -  Doc file 'inputs/index.md' contains an unrecognized relative link '../custom_ui_extensions/custom_hook', it was left as is. Did you mean '../custom_ui_extensions/custom_hook.md'?
INFO    -  Doc file 'inputs/index.md' contains an unrecognized relative link '../advanced/sub_description', it was left as is. Did you mean '../advanced/sub_description.md'?
```

Reply:
fixed